library/consul

```diff
@@ -6,19 +6,19 @@ GitRepo: https://github.com/hashicorp/docker-consul.git
 GitCommit: 74be3d0c6ce2e12805e0c9be4cf02a44e6f3cec1
 Directory: 0.X
 
-Tags: 1.10.3, 1.10, latest
+Tags: 1.10.4, 1.10, latest
 Architectures: amd64, arm32v6, arm64v8, i386
 GitRepo: https://github.com/hashicorp/docker-consul.git
 GitCommit: e677d8b7151a4e3a994741752b39ae42604cf97d
```

Comment:
None of these include any Dockerfile changes. I think the `GitCommit's need to be updated.

Reply:
whoops. derp. I was trying to reset around some other rogue diff and forgot to reapply. Thanks. Should be fixed now.